recipes/dynare_preprocessor/meta.yaml

```
@@ -0,0 +1,40 @@
+package:
+  name: dynare_preprocessor
+  version: 0.0.1
+
+source:
+  url: https://git.dynare.org/Dynare/preprocessor/-/archive/6235e6907526d468e2c35f14c38db1e308698bc9/preprocessor.tar.gz
+  sha256: 06f76245ff5c6ac0811438cea6b817a321dc702c188b680e2a3191b54837c4ab
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - '{{ compiler("cxx") }}'
+    - meson
+    - bison
+    - flex
+  host:
+    - git-lfs
+    - gsl
+    - openlibm
+    - boost-cpp
+    - boost
```

```suggestion
    - libboost-devel
```
Unless you need an old version of boost, the development package now has this name. If you need python, then list it explicitly instead of depending on the "boost" package (which depends on python).

---

Actually, I tried with `libboost-headers`, and got an error so I reverted to `libboost-devel`. Is that OK?